content/dstu2/openid-connect.md

```diff
@@ -0,0 +1,91 @@
+---
+title: OpenID Connect | DSTU 2 API
```

Comment:
Should we clarify userInfo support (or lack of): http://docs.smarthealthit.org/ (Simple Sign-in section)